.mailmap

```diff
@@ -9,6 +9,7 @@
 Aaron Bassett <abassett@gmail.com>
 Abhishek Lekshmanan <abhishek.lekshmanan@ril.com> <abhishek.lekshmanan@gmail.com>
```

Comment:
In any case only one of them will be taken into account. There is no way to resolve this so that past contributions stay with the previous mapping. There would need to be a time range for which a mailmap entry is valid, but that's not supported. In other words, unless you change your personal email, contributions from the past will be remapped incorrectly when using a recent .mailmap. There always is the option of using the former .mailmap to get the correct affiliation, which is probably the right thing to do.